README.md

```diff
@@ -120,3 +120,32 @@ The design philosophy of this class is heavily inspired by the trace event forma
 
 The main goals of this logger helper are clarity of output and simplicity for the developer. While performance is a nice-to-have, it is not the main concern of this helper. A minor performance impact compared to simply logging the events is to be expected.
 
+## Running
+
+The jar needs to be loaded for it to run. Assuming the jar is called `trace-event-logger-x.y.z.jar`, one would need to first build their java application with the jar in the *classpath* such as 
+
+```console
+user@host:~$ javac -cp trace-event-logger-x.y.z.jar my-project
+```
+	
+`-cp`: loads the jar
+
+and then, run it as 
+
+```console
+user@host:~$ java -Djava.util.logging.config.file=path/to/logging.properties -p bin:path/to/trace-event-logger-x.y.z.jar -m pacakge/path/to.Main
+```
+
+`-Djava.util.logging.config.file`: the path of the `logging.properties`
+
+`-p`: the packages, the location of the classes and the external logging jar
+
+`-m`: is the location of the main module to run.
```

Comment:
Why not `the location of the main module to run` as the other parameters are described declaratively ?

Reply:
done